frontend/lib/identification/verification_workflow/dialogs/positive_verification_result_dialog.dart

```diff
@@ -28,19 +53,43 @@ class PositiveVerificationResultDialog extends StatelessWidget {
       builder: (result, {refetch, fetchMore}) {
         final data = result.data;
         final region = result.isConcrete && data != null ? regionsQuery.parse(data).regionsByIdInProject[0] : null;
+        final bool isUncheckedStaticQrCode = !isChecked && widget.hasStaticVerificationCode;
         return InfoDialog(
-          title: localization.positiveVerificationDialogTitle,
-          icon: Icons.verified_user,
-          iconColor: Colors.green,
-          child: IdCard(
-            cardInfo: cardInfo,
-            region: region != null ? Region(region.prefix, region.name) : null,
+          title: isUncheckedStaticQrCode ? "Prüfung notwendig" : localization.positiveVerificationDialogTitle,
+          icon: isUncheckedStaticQrCode ? Icons.info : Icons.verified_user,
+          iconColor: isUncheckedStaticQrCode ? Colors.yellow : Colors.green,
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              Flexible(
+                child: IdCard(
+                  cardInfo: widget.cardInfo,
+                  region: region != null ? Region(region.prefix, region.name) : null,
+                ),
+              ),
+              if (widget.hasStaticVerificationCode)
+                Flexible(
+                  child: Padding(
+                    padding: const EdgeInsets.only(top: 16.0),
+                    child: CheckboxListTile(
+                      title: const Text('Ich habe die Daten mit einem amtlichen Lichtbildausweis abgeglichen.'),
+                      checkColor: Theme.of(context).primaryColor,
```

Comment:
Please don't adjust the checkbox color here (it looks weird on light theme)
You can adjust the checkboxTheme for the darkmode in themes.dart
Under `theme.copyWith(...)` add something like:

    checkboxTheme: CheckboxThemeData(
      checkColor: const MaterialStatePropertyAll(Colors.white),
      fillColor: MaterialStatePropertyAll(primaryColor),
    ),